server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLNonNull } from 'graphql';
+import GraphQLJSON from 'graphql-type-json';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { Account } from '../interface/Account';
+import { Unauthorized, Forbidden } from '../../errors';
+
+const accountMutations = {
+  editAccountSettings: {
+    type: new GraphQLNonNull(Account),
```

Comment:
Is that best practice vs returning the settings?

Reply:
If we just return the `settings` object we'll need to update Apollo's cache manually. By returning the full account Apollo will be able to update it automatically.

I have no strong opinion on which one is cleaner, but I like the idea that any returned entity should have a unique identifier. In the case of `settings`, its identifier is the account id.